target/linux/ramips/Makefile

```diff
@@ -11,6 +11,7 @@ SUBTARGETS:=mt7620 mt7621 mt76x8 rt288x rt305x rt3883
 FEATURES:=squashfs gpio
 
 KERNEL_PATCHVER:=5.15
+KERNEL_TESTING_PATCHVER:=6.1
```

Comment:
I realize this is a WIP but could `KERNEL_TESTING_PATCHVER` be added only for mt7621 subtarget?